sale_order_line_variant_description/views/product_view.xml

```diff
@@ -8,14 +8,12 @@
         <field name="inherit_id" ref="product.product_normal_form_view" />
         <field name="arch" type="xml">
             <field name="description_sale" position="attributes">
-                <attribute
-                    name="attrs"
-                >{'invisible': [('product_variant_count', '&gt;', 1)]}</attribute>
+                <attribute name="invisible">product_variant_count > 1</attribute>
             </field>
             <field name="description_sale" position="after">
                 <field
                     name="variant_description_sale"
-                    attrs="{'invisible': [('product_variant_count', '=', 1)]}"
+                    invisible="[('product_variant_count', '=', 1)]"
```

Comment:
Not migrated correctly, it should be a Python expression instead of a ORM domain.